### PR TITLE
[WIP] investigating double Activity export

### PR DESCRIPTION
**Pull request by TimothyMothra**
I'm investigating an issue where a single Activity is intermittently exported twice. https://github.com/Azure/azure-sdk-for-net/pull/31567

This doesn't happen on my local box but does occur on almost every build on Azure's build servers.
I want to run this over here and see if it repros.

### CODE REVIEW

File: test/OpenTelemetry.Tests/E2ETracesTest.cs [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+            using var activitySource = new ActivitySource(activitySourceName);
+
+            var logCategoryName = $"logCategoryName{uniqueTestId}";
+
+            List<Activity> exportedActivities = new List<Activity>();
+            List<LogRecord> exportedLogs = new List<LogRecord>();
+
+            var tracerProvider = Sdk.CreateTracerProviderBuilder()
+                .AddSource(activitySourceName)
+                .AddInMemoryExporter(exportedActivities)
```
[the remark this answers is not in the document]
Comment (TimothyMothra) [reply test/OpenTelemetry.Tests/E2ETracesTest.cs:59]: Thanks. Just double checked and confirmed I'm using SimpleActivityExportProcessor in my tests.

This exact test has an issue in the Azure repo. (linked above)

**Comment by TimothyMothra** [comment]
Closing this for now. 
I was able to reproduce this with a test using only `ActivityListener` and with no OpenTelemetry components.

**Comment by cijothomas** [comment]
> Closing this for now. I was able to reproduce this with a test using only `ActivityListener` and with no OpenTelemetry components.

Could you open issue in runtime repo and share a link here?

**Comment by TimothyMothra** [comment]
> > Closing this for now. I was able to reproduce this with a test using only `ActivityListener` and with no OpenTelemetry components.
> 
> Could you open issue in runtime repo and share a link here?

Here: https://github.com/dotnet/runtime/issues/76757

**Pull request closed without merging**